IaC/AKS/README.md

```diff
@@ -486,21 +484,35 @@ Deploy Web Validate to drive consistent traffic to the AKS cluster for monitorin
 
 ```bash
 
-cd $REPO_ROOT/IaC/AKS/cluster/charts/smoker
+cd $REPO_ROOT/IaC/AKS/cluster/charts
+
+cp ./smoker/helm-config.example.yaml ./smoker/helm-config.yaml
 
 kubectl create namespace ngsa-smoker
-kubectl create secret generic ngsa-smoker-secrets \
-  --namespace ngsa-smoker \
-  --from-literal=WorkspaceId=$(az monitor log-analytics workspace show -g $Ngsa_Log_Analytics_RG -n $Ngsa_Log_Analytics_Name --query customerId -o tsv) \
-  --from-literal=SharedKey=$(az monitor log-analytics workspace get-shared-keys -g $Ngsa_Log_Analytics_RG -n $Ngsa_Log_Analytics_Name --query primarySharedKey -o tsv)
+helm install ngsa-smoker smoker -f ./smoker/helm-config.yaml --namespace ngsa-smoker
 
-cp helm-config.example.yaml helm-config.yaml
+# Verify the pods are running
+kubectl get pods --namespace ngsa-smoker
+
+```
+
+## Fluent Bit Log Forwarding
+
+Deploy Fluent Bit to forward application and smoker logs to the Log Analytics instance.
+
+```bash
 
 cd $REPO_ROOT/IaC/AKS/cluster/charts
 
-helm install ngsa-smoker smoker -f ./smoker/helm-config.yaml --namespace ngsa-smoker
+kubectl create namespace fluentbit
+kubectl create secret generic fluentbit-secrets \
+  --namespace fluentbit \
+  --from-literal=WorkspaceId=$(az monitor log-analytics workspace show -g $Ngsa_Log_Analytics_RG -n $Ngsa_Log_Analytics_Name --query customerId -o tsv) \
+  --from-literal=SharedKey=$(az monitor log-analytics workspace get-shared-keys -g $Ngsa_Log_Analytics_RG -n $Ngsa_Log_Analytics_Name --query primarySharedKey -o tsv)
 
-# Verify the pods are running
-kubectl get pods --namespace ngsa-smoker
+helm install fluentbit fluentbit -f ./fluentbit/helm-config.yaml --namespace fluentbit --set logSuffix=$Ngsa_Env
```

Comment:
do we want to use a log suffix or will our logs be in different Log Analytics instances?